src/main/java/task/Deadline.java

```diff
@@ -0,0 +1,23 @@
+package task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    private LocalDate by;
+
+    public Deadline(String name, LocalDate by) {
+        super(name);
+        this.by = by;
+    }
+
+    @Override
+    public String taskTitle() {
```

Comment:
Should this method name be modified to contain a verb?
```suggestion
    public String getTaskTitle() {
```